config/environments/production.rb

```diff
@@ -75,10 +75,6 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  unless config.x.disable_password
-    config.x.app_password = ENV.fetch('SCHEMA_REGISTRY_PASSWORD')
```

Comment:
This feels a bit dangerous to me since we'd want this to blow up in production if `SCHEMA_REGISTRY_PASSWORD` wasn't set. How bad is it to specify `SCHEMA_REGISTRY_PASSWORD=avro` when running locally?